.travis.yml

```diff
@@ -14,12 +14,16 @@ before_install:
   - nvm install node
 
 env:
-  - DB=pg
+  global:
+    - DB=pg
+    - secure: mGy9AGSIv/OvNspycY9ghSOp9/g6wZdrHGG9l3TmAvPPQJmMn+MLnDcooX3pGKENxGIslF/5f1QX0b2ouoRXQu5jHj6EkrWjpqQE/E0xWpxhFjXoo/fE/tTFpAZKxme4wcSuFxeqfVEQ6Rk07BSG2c609AbODxnxywTaTH9rfob5HaSD+uP4E3MCPxZ6SK941pqEkfwNI9Z9g5C/oow4jn9BHP2Dy1hTzwMkfBGIsjRFzvjm7Ey4h19Zbt9YKfzjo1ZzPQ4q0ensqNJEIbryCTC/xZPl8KXA2EKwnMjDnfrhhZsX7g8FdyCs8AaeCDbCXpgXKatWsSxcjxUf6NTAg7oS7w4N/EyJguyXMROnHl+EMKSWhq74xa3dCEu5sedN5KJicHl2l1wJKKQCiZ9yZ5j86p5hqqU/B3kv9fspMuqcCIO3S4xOAg8Jv5dk0oDpnhbrWhteVryEUtwlSdBoup+MvXjbO7auBVfdx7luugBvEkT/qCAhXt4ajVJ8lQuYIJ38UKDXkblgkcjfqKG0HFTkNfOQcuxE7XoVYSXH+lxH9U3DYQeEuMH1P57ADXmUzFFEoJxjVsSfUdsFWFLK7Xl6jc6imjFxLLw8J4AmEvdyexcetZ33rl4frUhGwTsnV4cIPKcd9dlclDy5/NGKuf5gVsGNcoQgsvO1YLOKGAc=
+    - secure: uj2OXO5OTNCHA/byiP0o5aXHLeF9TItHUObtuzkS2ZbbTtS38r9DPnAEkIOToSkKXeiWd4xAX2Cgql5be7JwerxwYFJo75ijHZrnBLI97LL2Kq12BEZFBbobCSKHxxSAAa9U9wcUtypb41OsFaJShvD5H7Jh11JwirHRKG6IpQzTdtEzzFXl+Vg3h0UwGdr45TtQhnent3sq636TrQV5zNN1ggr5Qk+TXrrxbH+bn92QViPxoT/U1M3qHZYplWKo1S96wMPWc/x0aGhL/aASoLwcO7/Ach8PlmqYHF3mhqs7QmWMXh8OqS0HX/xRAd5OD6BjZJ6cseAa1y+tNSQRroi8pr2QsyKu7Ji1eKHVuIpYYVfCNor0YfwKtK3b+G7iU7TUL5cdfno2NGU90/9ZkWTtSTX+UOPCmkuKV9FK88P8r0+N7hr881PS/5BtpnWJgkLtgOIDVrmOYBQpvW2qJNMYYLLPwz3BawUG8JYcEhj9P7y5NAZDSFro+0X9FdxjLWRbIdw+2WanhbMCfk53RxbUtQ5gsatTOy34ANaUyXWAmSS9ks+ZGr53WuOnV3yTnLOTXTlTB3nU2CgvTxLYGzPAfmoxLgh5RJi583ubZ4S+Q79ki/PFHD5WE3ZYt36ul+d0Y9A9IX9F0ex1ved4ZRXz7TnsZFJrVAbI/TDnD1Y=
+    - secure: atEtVos72LKeP2RkvJe3Kv775yK8YkbzdzVBv03KSWkHNqxMLFxxY03J1rJpWWyW+jEGZUjYGKtH9WIWMXqJOHkIqVkbxtvHerDBjL51sWndU6h0YQgySwXilH6nY6oWXL4ul+N/nYUiEF1sJwHDr04tUMJws7707gbZsNdN4EWELfcfpSm7GyhhPKRMNoIj1LPHrEx690EnpypGAUbM4zx55wXP8hfbXxULF+dWg43/GQVgBwi0xVxhfS17c3hMfa4Ohi8j7EacNuBcTNQLIxUcpzlNxGDpLTUfkHWNZ4soBJn6ZMyNO79Far+CsbKhqe0K08Y9wIgrz741ORixAWj7x/10Al9Q7qQ3i7R0m2ZtIbhVmFc9lYAs3U/HQI08OIOOo3uaOvwQxJDFwCqRKL1ckRfZ5Wz8ax2hT75r9w2hiJQ32RNJHswQGPEhwe0jV5Q5krumdAnI7DdRt8JbO8neugeFa4gzVPc9P1tRzZ2n755ckaGRoSu18YqxX9wUL+dIzoY9vh6r8Bkv1eaNi4kTeWl8Cy1l0ObdcpzYcsCMngDdVpmsFyr3fd3i9+dRkPQLXEaK38T6PakLA+y31I48w0lBhBCNKfi9Z1fRl92EXxlYuTqb4EdDlfxHV6GfngZVBkeNOdA5/P882SmYiQ88KF8Wlq/SD6NGW1Nuz6I=
 
```

Comment:
is it worth adding a couple of comments as to what the encrypted data is maybe in case we need to change it